### PR TITLE
Updating README with the latests command to run the Sinopia Editor.

### LINKED ISSUE #105: Update README for react homepage
For dev server:
`npm run dev-start`

For prod
`npm run build`
`npm start`

### PR DESCRIPTION
**Pull request by jermnelson**
Updated the documentation with the correct commands for building and running the Sinopia Editor with React and Docker. Moved `bfe` sections to later in the README. 

Leaving this **WIP** in case @SaravShah wants to add comments on the React portion of the README.

Fixes #105

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,15 @@
...
-From a design standpoint, the objective with `bfe` was to create the simplest
-'pluggable' form editor one can to maximize experimental implementer's abilities
-to create/edit BIBFRAME data.  The current focus is to transform bfe into a production ready tool.
 
 # Sinopia BIBFRAME Editor
 
-Technical documentation specific to the Sinopia BIBFRAME Editor may also be found in the [wiki](https://github.com/LD4P/sinopia_editor/wiki/Sinopia-Editor).
+Technical documentation specific to the Sinopia BIBFRAME Editor may also be found in the [wiki](https://github.com/LD4P/sinopia_editor/wiki/Sinopia-Editor). The
+[Sinopia Editor][GIT_REPO] is also available in staging at
+[stage.sinopia.io][staging]. The Sinopia Editor is a [React][REACT] application
```
Comment (ndushay) [inline README.md]: Is this true yet (stage.sinopia.io) ?   We don't have anything like a functional editor there yet, do we?

File: README.md [hunk 2]
```diff
@@ -1,24 +1,15 @@
...
-...
-From a design standpoint, the objective with `bfe` was to create the simplest
-'pluggable' form editor one can to maximize experimental implementer's abilities
-to create/edit BIBFRAME data.  The current focus is to transform bfe into a production ready tool.
 
 # Sinopia BIBFRAME Editor
 
-Technical documentation specific to the Sinopia BIBFRAME Editor may also be found in the [wiki](https://github.com/LD4P/sinopia_editor/wiki/Sinopia-Editor).
+Technical documentation specific to the Sinopia BIBFRAME Editor may also be found in the [wiki](https://github.com/LD4P/sinopia_editor/wiki/Sinopia-Editor). The
+[Sinopia Editor][GIT_REPO] is also available in staging at
```
Comment (ndushay) [inline README.md]: looks like you put in some hard returns ... do you want it to be that way, or responsively formated, or?

File: README.md [hunk 3]
```diff
@@ -1,24 +1,15 @@
...
+[stage.sinopia.io][staging]. The Sinopia Editor is a [React][REACT] application
+with all new user interfaces and functionality using React and the React
+ecosystem. Portions of the codebase originally extracted from the Library of
+Congress `bfe` project.
```
Comment (ndushay) [inline README.md]: i would put in a link to their bfe project.

File: README.md [hunk 4]
```diff
@@ -31,45 +22,50 @@ Technical documentation specific to the Sinopia BIBFRAME Editor may also be foun
 2.  Install [npm](https://www.npmjs.com/)
 3.  Run `npm init`, and follow the instructions that appear.
 4.  Get latest npm: `npm install -g npm@latest`
-5.  Run `npm install grunt-cli` to install grunt-cli.
-6.  Run `npm install`. This installs everything needed for the build to run successfully.
-7.  Run `npm run build` to build the code.  (TEMP: Run `npm run grunt` or `grunt` to build the BFE code.)
+5.  Run `npm run build` to build the code.
+
+### Installing BFE code
+1.  Run `npm install grunt-cli` to install grunt-cli.
```
Comment (ndushay) [inline README.md]: do we want to just add grunt-cli to the dev dependencies?  Would make the instructions simpler.

File: README.md [hunk 5]
```diff
@@ -31,45 +22,50 @@ Technical documentation specific to the Sinopia BIBFRAME Editor may also be foun
 2.  Install [npm](https://www.npmjs.com/)
 3.  Run `npm init`, and follow the instructions that appear.
 4.  Get latest npm: `npm install -g npm@latest`
-5.  Run `npm install grunt-cli` to install grunt-cli.
-6.  Run `npm install`. This installs everything needed for the build to run successfully.
-7.  Run `npm run build` to build the code.  (TEMP: Run `npm run grunt` or `grunt` to build the BFE code.)
+5.  Run `npm run build` to build the code.
```
Comment (ndushay) [inline README.md]: does webpack build take care of the npm install as well?  if so, cool!

File: README.md [hunk 6]
```diff
@@ -31,45 +22,50 @@ Technical documentation specific to the Sinopia BIBFRAME Editor may also be foun
...
 `npm start`
 
 Follow installation instructions, then run `npm start` or `node server.js` to start the web server using Express.
-This will start up the code at http://localhost:8000.
-
-TEMP: The Sinopia Editor code is currently available via http://localhost:8000/bfe-index.html.
+This will start up the code at [http://localhost:8000](http://localhost:8000).
 
-TEMP:  Note that `index.html` is meant to show the Sinopia home page via React.
+The Sinopia Editor code is currently available via [stage.sinopia.io][staging]
```
Comment (ndushay) [inline README.md]: ah ha!   Should this be with the other reference to it above?  Or not?

File: README.md [hunk 7]
```diff
@@ -31,45 +22,50 @@ Technical documentation specific to the Sinopia BIBFRAME Editor may also be foun
...
+- React components are located in `src/components/` directory
 
 ### Run the server with webpack-dev-webserver
 
 `npm run dev-start`
 
-This runs the webpack-dev-server, allowing immediate loading of live code changes without having to restart the server.
+Runs the webpack-dev-server, allowing immediate loading of live code changes
+without having to restart the server. The webpack-dev-server is available on at
+[http://localhost:8080](http://localhost:8080).
```
Comment (ndushay) [inline README.md]: same deal with hard returns -- are you sticking with an 80 char line or something like that?

File: README.md [hunk 8]
```diff
@@ -198,17 +223,23 @@ Contributors
...
+* **LD4P2 Sinopia Project Team**
+  * [GitHub](https://github.com/ld4p/)
 
 
 License
 -------
 
-Unless otherwise noted, code that is original to `bfe` is in the Public Domain.
+Unless otherwise noted, code that is original to the `Sinopia Editor` is licensed
+under the [Apache 2](https://www.apache.org/licenses/LICENSE-2.0).
```
Comment (ndushay) [inline README.md]: uh - is this true?  Do we need to change the LICENSE.txt file?   I'm all for whatever is the correct thing - let's just be correct in all places.  Stanford license vs LD4P vs LC ... ugh.

Comment (atz) [reply README.md]: `package.json` references ISC and MIT licenses.

File: README.md (hunk 5 above)
Comment (jermnelson) [reply README.md]: I don't think the `npm run build` has the `npm install` step although we could add it as part of the `build` script in the package.json.

File: README.md (hunk 6 above)
Comment (jermnelson) [reply README.md]: I adjusted the reference in earlier section.

**Pull request merged**